bdd100k/label/to_mask.py

```diff
@@ -172,18 +211,40 @@ def semseg_to_masks(
                 continue
 
             category_id = cat_name2id[label.category]
-            color = np.array([category_id])
+            if mode in ["sem_seg", "drivable"]:
+                color = np.array([category_id])
+            else:
+                color = set_lane_color(label, category_id)
             colors.append(color)
             poly2ds.append(label.poly_2d)
 
-        colors_list.append(colors)
-        poly2ds_list.append(poly2ds)
-
+    logger.info("Start Conversion for Seg to Masks")
     frames_to_masks(
-        nproc, out_paths, colors_list, poly2ds_list, with_instances=False
+        nproc,
+        out_paths,
+        colors_list,
+        poly2ds_list,
+        with_instances=False,
+        back_color=back_color,
+        closed=closed,
     )
 
 
+ToMasksFunc = Callable[[List[Frame], str, bool, bool, int], None]
+semseg_to_masks: ToMasksFunc = partial(
+    seg_to_masks, mode="sem_seg", back_color=IGNORE_LABEL, closed=True
+)
+drivable_to_masks: ToMasksFunc = partial(
+    seg_to_masks,
+    mode="drivable",
+    back_color=len(drivables) - 1,
+    closed=True,
+)
+lanemark_to_masks: ToMasksFunc = partial(
```

Comment:
lane_mark to mask draw non-closed polygons.
Other tasks to_mask functions draw closed polygons